packages/storybook-html/src/components/core/colour-apply.story.mdx

```diff
@@ -35,17 +35,20 @@ The purple gradient is already incorporated in the header, footer and other comp
     <th class="uqds-table__heading">Example</th>
     <th class="uqds-table__heading">Name</th>
     <th class="uqds-table__heading">Variable</th>
+    <th class="uqds-table__heading">HEX</th>
     <th class="uqds-table__heading">Usage</th>
   </tr>  <tr  class="uqds-table__row">
     <td class="uqds-table__item"><div class="uqds-colour__palette uqds-colour--purple"></div></td>
     <td class="uqds-table__item" data-title="Name:">UQ Purple</td>
     <td class="uqds-table__item" data-title="Variable:"><code>$uq-purple</code></td>
+    <td class="uqds-table__item" data-title="HEX:">#51247A</td>
     <td class="uqds-table__item" data-title="Usage:">Reflect UQ brand in accents and emphasis</td>
   </tr>
   <tr  class="uqds-table__row">
     <td class="uqds-table__item"><div class="uqds-colour__palette uqds-purple-gradient"></div></td>
     <td class="uqds-table__item" data-title="Name:">Purple Gradient</td>
```

Comment:
Shouldn't this be "UQ Purple Gradient" instead of just "Purple Gradient" for consistency

Reply:
We followed the Brand Guidelines and how it is titled